src/main.js

```diff
@@ -1953,6 +1966,17 @@ ehDownloadAction.innerHTML = ehDownloadArrow + ' <a>Download Archive</a>';
 ehDownloadAction.addEventListener('click', function(event){
 	event.preventDefault();
 
+	if (setting['play-silent-music']) {
+    oscillator = audioCtx.createOscillator()
+    gainNode = audioCtx.createGain()
+    oscillator.connect(gainNode)
+    gainNode.connect(audioCtx.destination)
+    oscillator.frequency.value = 1
+    gainNode.gain.value = 0.001
+    oscillator.start()
+	}
```

Comment:
Please use tab to indent instead of 2-spaces, also add a semicolon (`;`) after each line.